src/main/java/seedu/address/model/event/Event.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.event;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
```

Comment:
Add class-level javadocs to tell us about the class

Reply:
Done-by Sol